apps/bandwagon/forms.py

```diff
@@ -136,6 +135,9 @@ class CollectionForm(ModelForm):
     icon = forms.FileField(label=_lazy(u'Icon'),
                            required=False)
 
+    # This is just a honeypot field for bots to get caught
+    your_name = forms.CharField(widget=forms.HiddenInput(), required=False)
```

Comment:
In my experience it doesn't make any big difference for all the very dumb bots but assistive tech is a good point. `display:none;` should cover this case completely too, does it not?